### PR TITLE
specify version and platfrom variables for gcloud tool

### PR DESCRIPTION
**Pull request by krhubert**

### CODE REVIEW

**Comment by rfhayashi** [comment]
@krhubert, I don't know why I didn't received a notification e-mail of your pull request. Just saw it today. We need to fix the test before merging. Just change the test to check if the new default version is installed.

File: README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ Example Playbook
 
     - hosts: servers
       roles:
-         - { role: rfhayashi.gcloud }
+         - { role: ansible-gcloud }
```
Comment (rfhayashi) [inline README.md]: I think we should keep `rfhayashi.gcloud` as it is the way the role is published on Ansible Galaxy. Can you revert that?

Comment (krhubert) [inline README.md]: Sure, my mistake, doing

**Comment by krhubert** [comment]
Done

**Comment by rfhayashi** [comment]
Thank you! Would you mind documenting the new variables in the readme file? I'll merge afterwards.

File: defaults/main.yml [hunk 2]
```diff
@@ -1,2 +1,5 @@
 ---
 # defaults file for ansible-gcloud
+google_cloud_sdk_version: 136.0.0
+google_cloud_sdk_platfrom: linux-x86_64
```
Comment (rfhayashi) [inline defaults/main.yml]: Just realized after merging that there is a typo here. Can you fix that?

ps: it's also wrong in the readme

Comment (rfhayashi) [inline defaults/main.yml]: Had some time free and did it myself, thanks!

**Pull request merged**